packages/server/src/api/export/export.sql.controller.ts

```diff
@@ -0,0 +1,74 @@
+import moment from 'moment-timezone';
+import { Response } from 'express';
+import { json2csvAsync } from 'json-2-csv';
+import { ExportOrgQuery } from '@covid19-reports/shared';
+import {
+  ApiRequest,
+  OrgParam,
+} from '../api.router';
+import { Observation } from '../observation/observation.model';
+import { Roster } from '../roster/roster.model';
+import { RosterHistory } from '../roster/roster-history.model';
+import { Unit } from '../unit/unit.model';
+
+class ExportSqlController {
+
+  async exportOrgToCsv(req: ApiRequest<OrgParam, null, ExportOrgQuery>, res: Response) {
+    const startDate = moment(req.query.startDate ?? 0);
+    const endDate = moment(req.query.endDate);
+    const orgId = req.params.orgId;
+    const units: Unit[] = await Unit.createQueryBuilder('unit')
+      .select('unit.name')
+      .where(`unit.org_id = :orgId`, { orgId })
+      .getRawMany();
+
+    const unitNames: string[] = units.map(unit => unit.name);
```

Comment:
Aren't unit names already returned on line 20 with the `select('unit.name')` query?